pkg/apis/seedmanagement/v1alpha1/conversions.go

```diff
@@ -54,10 +54,10 @@ func Convert_seedmanagement_GardenletConfig_To_v1alpha1_GardenletConfig(in *seed
 	if err := autoConvert_seedmanagement_GardenletConfig_To_v1alpha1_GardenletConfig(in, out, s); err != nil {
 		return err
 	}
-	if out.Config.Raw == nil {
+	if out.Config.Raw == nil && out.Config.Object != nil {
```

Comment:
I think the reason this was added was because of a failing test but its need does not seem related to the issue. Is this an unrelated bug that we just haven't hit until this moment?

Reply:
I think it make sense to keep the newly added `&& out.Config.Object != nil ` part. Indeed, it seems like a existing nil pointer that was present.